cmdr/src/bin/giti.rs

```diff
@@ -67,7 +68,8 @@ pub fn launch_giti(cli_arg: CLIArg) {
     match try_run_command(cli_arg) {
         // Command ran successfully.
         Ok(_) => {
-            giti_ui_templates::show_exit_message();
+            // TODO: uncomment the line below?
```

Comment:
Using `cli_arg` you can get the command that the user passed via the command line. And this can be used to decide which commands should get this exit message and which ones should not.

Reply:
I used matches! macro

```
            if matches!(
                cli_arg.command,
                CLICommand::Branch {
                    command_to_run_with_each_selection: Some(BranchSubcommand::Delete),
                    ..
                }
            )
```